cdk-v2/Vagrantfile

```diff
@@ -9,6 +9,10 @@
 # * Have valid RH employee subscription account
 # * Have 'vagrant-adbinfo' plugin installed
 
+Vagrant.require_version ">= 1.7.4"
```

Comment:
higher than 1.72 you mean?
Problematic for me since the official package for FC21 is 1.72